test/framework/src/test/java/org/elasticsearch/search/internal/ConcurrentSearchSingleNodeTests.java

```diff
@@ -17,6 +18,7 @@
 
 import java.io.IOException;
 
+@LuceneTestCase.AwaitsFix(bugUrl = "todo: this is somewhat meaningless with without a worker pool")
```

Comment:
Do we even want to keep tests like these? We have some others that check that parallelization actually happens, 
what does it even mean that the min docs per slice jumps to 50k if concurrent search is disabled? :)

Reply:
I think this was a way to verify ESSingleNodeTestCase behaviour, see #97740 . It's to check that when concurrency is enabled in tests, we generate as many slices as possible.